src/libraries/Microsoft.Extensions.Logging.Console/src/SimpleConsoleFormatter.cs

```diff
@@ -15,10 +15,13 @@ internal sealed class SimpleConsoleFormatter : ConsoleFormatter, IDisposable
         private const string LoglevelPadding = ": ";
         private static readonly string _messagePadding = new string(' ', GetLogLevelString(LogLevel.Information).Length + LoglevelPadding.Length);
         private static readonly string _newLineWithMessagePadding = Environment.NewLine + _messagePadding;
-        private static readonly bool _isAndroidOrAppleMobile = RuntimeInformation.IsOSPlatform(OSPlatform.Create("ANDROID"))
-                                                            || RuntimeInformation.IsOSPlatform(OSPlatform.Create("TVOS"))
-                                                            || RuntimeInformation.IsOSPlatform(OSPlatform.Create("IOS"))
-                                                            || RuntimeInformation.IsOSPlatform(OSPlatform.Create("MACCATALYST"));
+#if NETCOREAPP
+        private static readonly bool _isAndroidOrAppleMobile => OperatingSystem.IsAndroid()
+                                                            || OperatingSystem.IsTvOS()
+                                                            || OperatingSystem.IsIOS(); // returns true on MacCatalyst
+#else
+        private const bool _isAndroidOrAppleMobile = false;
```

Comment:
```suggestion
        private static bool IsAndroidOrAppleMobile => OperatingSystem.IsAndroid()
                                                            || OperatingSystem.IsTvOS()
                                                            || OperatingSystem.IsIOS(); // returns true on MacCatalyst
#else
        private static bool IsAndroidOrAppleMobile => false;
```

Reply:
For my knowledge - why is 

```
private static bool IsAndroidOrAppleMobile => false;
```

preferred over

```
private const bool IsAndroidOrAppleMobile = false;
```

?

Reply:
Either one is fine. I think it looks better when both sides of the ifdef define the same thing.

The point of my suggestion was to fix the name so that it follows naming conventions.